Dockerfile

```diff
@@ -46,6 +46,10 @@ RUN apt-get update \
    jellyfin-ffmpeg \
    openssl \
    locales \
+   unzip \
+# install Noto CJK fonts
+ && wget https://noto-website-2.storage.googleapis.com/pkgs/NotoSansCJK-Bold.ttc.zip -O /usr/share/fonts/NotoSansCJK-Bold.ttc.zip \
```

Comment:
This may lead to another problem:   I'm not sure if people in China mainland can access the url without any proxy helped.

Reply:
I think we do not need to access this url directly. The access to the built image is enough.